### PR TITLE
Support ONNX custom exporter like mmdeploy

### PR DESCRIPTION
**Pull request by take-cheeze**

### CODE REVIEW

**Comment by take-cheeze** [comment]
Seems some warning from protobuf:
```
00:10:14.826969 STDOUT 1390] | /opt/pyenv/versions/3.12.7/lib/python3.12/site-packages/google/protobuf/internal/well_known_types.py:91: in <module> |  
-- | -- | --
00:10:14.827014 STDOUT 1390] | _EPOCH_DATETIME_NAIVE = datetime.datetime.utcfromtimestamp(0) |  
00:10:14.827016 STDOUT 1390] | E   DeprecationWarning: datetime.datetime.utcfromtimestamp() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.fromtimestamp(timestamp, datetime.UTC).
```

**Comment by take-cheeze** [comment]
:naruhodo: https://github.com/protocolbuffers/protobuf/issues/17856

**Comment by take-cheeze** [comment]
https://ci.preferred.jp/pytorch-pfn-extras.torch204-linux/179532/#L833 somehow old protobuf is installed

File: pytest.ini [hunk 1]
```diff
@@ -41,6 +41,8 @@ filterwarnings =
     ignore:`torch.cpu.amp.autocast\(args...\)` is deprecated.:FutureWarning
     # For torch >= 2.4: Unnecessary warnings on import of onnxruntime in CI environments.
     ignore:Unsupported Windows version \(2016server\).:UserWarning
+    # For custom exporter test in onnx and mmdeploy
+    ignore:.*MMCV.*:UserWarning
```
Comment (take-cheeze) [inline pytest.ini:45]:
```suggestion
```

**Review by linshokaku — APPROVED** [review]
Thank you so much for adding the new feature!

**Pull request merged**